.github/workflows/container-version.yaml

```diff
@@ -0,0 +1,25 @@
+on:
+  schedule:
+    - cron: '37 13 * * *'
+name: busybox-update workflow
+jobs:
+  checkVersionAndCreatePR:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run busybox updater
+        run: |
+          chmod +x ./scripts/busybox-updater.sh
+          ./scripts/busybox-updater.sh
+        shell: bash
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          signoff: true
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: 'Updates busybox SHA'
+          title: '[busybox] Updates busybox SHAs'
+          body: >
+            This PR is auto-generated by 
+            [create-pull-request](https://github.com/peter-evans/create-pull-request).
+          labels: busybox, automated pr
```

Comment:
Looks like we can add `team-reviewers` as well, maybe it would be worth to add Thanos maintainers or so, so we make sure these PRs do not slip through?

Reply:
We will need a private access token to enable this.

Reply:
Please add a new line here.